### PR TITLE
CORE-7074 - move to official micrometer release

### PR DESCRIPTION
**Pull request by driessamyn**
Remove dependency on R3 compiled version of Micrometer, since our patch was merged in the official version.

### CODE REVIEW

**Comment by ronanbrowne** [comment]
Finally :wink:

**Comment by driessamyn** [comment]
> Finally 😉

Yes, if it had been released 48h earlier we wouldn't have had to release our own :( 

Sorry for putting a bunch of work on you only for a single release. 🍻 on me next time ;)

**Review by ronanbrowne — DISMISSED** [review]
LGTM

**Review by yarivus — DISMISSED** [review]
LGTM

File: buildSrc/src/main/groovy/corda.quasar-app.gradle [hunk 1]
```diff
@@ -17,7 +17,7 @@ quasar {
         'com.networknt.schema**',
         'com.typesafe.**',
         'de.javakaffee.kryoserializers**',
-        'net.corda.micrometer.**',
+        'io.micrometer:micrometer.**',
```
Comment (chrisr3) [inline buildSrc/src/main/groovy/corda.quasar-app.gradle]: This is supposed to be a package glob, not a Maven coordinate. Something like:
```
io.micrometer.**
```

**Review by chrisr3 — CHANGES_REQUESTED** [review]

File: buildSrc/src/main/groovy/corda.quasar-app.gradle (hunk 1 above)
Comment (driessamyn) [reply buildSrc/src/main/groovy/corda.quasar-app.gradle]: Thanks for spotting this, @chrisr3 
I've changed it now.

**Review by chrisr3 — APPROVED** [review]
:+1:

**Review by yarivus — APPROVED** [review]
LGTM

**Pull request merged**